docs/els-for-runtimes-and-libraries/rollup/README.md

```diff
@@ -0,0 +1,156 @@
+
+   <TableTabs label="Choose Rollup version: " >
+
+     <template #rollup_2.79.1>
+
+     <CodeWithCopy>
+
+     ```text
+     "dependencies": {
+       "rollup": "npm:@els-js/rollup@2.79.1-tuxcare.2"
+     }
+     ```
+
+     </CodeWithCopy>
+
+     </template>
+
+     <template #rollup_2.79.2>
+
+     <CodeWithCopy>
+
+     ```text
+     "dependencies": {
+       "rollup": "npm:@els-js/rollup@2.79.2-tuxcare.2"
+     }
+     ```
+
+     </CodeWithCopy>
+
+     </template>
+
+   </TableTabs>
+
+5. You need to remove the `node_modules` directory and the `package-lock.json` file, and also clear the `npm cache` before installing the patched packages. Use the following commands:
+   
+   <CodeWithCopy>
+
+   ```text
+   rm -rf node_modules package-lock.json && npm cache clean --force
+   ```
+
+   </CodeWithCopy>
+
+6. Run the following command to install the ELS version of the Rollup library (token for the TuxCare repository will be automatically picked up from your `.npmrc` file):
+
+   <CodeWithCopy>
+
+   ```text
+   npm install
+   ```
+
+   </CodeWithCopy>
+
+   You will see an output like:
+
+   ```text
+   added 15 packages, and audited 16 packages in 2s
+
+   1 package is looking for funding
+   run `npm fund` for details
+
+   found 0 vulnerabilities
+   ```
+
+7. You've successfully installed the Tuxcare ELS version of the Rollup library into your project.
+
+## Vulnerability Exploitability eXchange (VEX) 
+
+VEX is a machine-readable format that tells you if a known vulnerability and is actually exploitable in your product. It reduces false positives, helps prioritize real risks.
+
+TuxCare provides VEX for Rollup ELS versions: [security.tuxcare.com/vex/cyclonedx/els_lang_javascript/rollup/](https://security.tuxcare.com/vex/cyclonedx/els_lang_javascript/rollup/).
+
+## How to Upgrade to a Newer Version of TuxCare Packages
+
+If you have already installed a package with a `tuxcare.1` suffix and want to upgrade to a newer release (for example, `tuxcare.3`), there are two options:
+
+* **Option 1**. Run the `npm install` command with the specific version. This will automatically update both `package.json` and `package-lock.json`:
+
+  <CodeWithCopy>
+
+  ```text
+  npm install rollup@npm:@els-js/rollup@2.79.2-tuxcare.3
+  ```
+
+  </CodeWithCopy>
+
+* **Option 2**. Update the version string in your `package.json`, remove installed files and clear npm cache to avoid conflicts:
+
+  <CodeWithCopy>
+
+  ```text
+  rm -rf node_modules package-lock.json && npm cache clean --force
+  npm install
+  ```
+
+  </CodeWithCopy>
+
+## Resolved CVEs
+
+Fixes for the following vulnerabilities are available in ELS for Rollup from TuxCare versions:
+
+| CVE ID         | CVE Type | Severity | Affected Libraries | Vulnerable Versions |
+| :------------: | :------: |:--------:|:------------------:| :----------------: |
+| CVE-2024-47068 | Direct   | Medium   |       rollup       | <2.79.2, <3.29.5, <4.22.4 |
```

Comment:
```suggestion
| CVE-2024-47068 | Direct   | Medium   |       rollup       | < 2.79.2, < 3.29.5, < 4.22.4 |
```

Reply:
otherwise we get a sweet emoji instead of the versions

<img width="755" height="221" alt="Снимок экрана от 2025-10-07 15-18-06" src="https://github.com/user-attachments/assets/741337c4-85fa-4fe2-ab75-550fd3f46739" />